Dependencies/AndroidExtensions/Include/AndroidExtensions/JavaWrappers.h

```diff
@@ -277,6 +291,15 @@ namespace android::view
 
         Display getDefaultDisplay();
     };
+
+    class Surface : public java::lang::Object
+    {
+    public:
+        Surface();
+
+        void initWithSurfaceTexture(android::graphics::SurfaceTexture surfaceTexture);
```

Comment:
Similar comment, convert this into a constructor to match the Java API?

Reply:
See previous comment.